neuralmonkey/encoders/recurrent.py

```diff
@@ -154,12 +170,15 @@ def __init__(self,
     @tensor
     def rnn_input(self) -> tf.Tensor:
         return dropout(self.input_sequence.temporal_states,
-                       self.dropout_keep_prob, self.train_mode)
+                       self.dropout_keep_prob,
+                       self.train_mode)
```

Comment:
proč? nikdo neřek, že má bejt jeden argument na jednu řádku

Reply:
osobni preference - kdyz mas skoro vse "1 radek ~ 1 argument" a pak najednou mas na radku dva argumentu, muze se snadno stat, ze to prehlednes

Reply:
Moje zásada je, že když už se to nevejde na jednu řádku, tak se odentruje hned za první zavorkou, a pak se všechny parametry píšou za sebe normálně do řady. Takhle nic neprehlidnes a ten kód pak není tak roztahanej. Něco jinýho je to u definici funkce, která žere milion argumentů a ještě tam jsou ty typy. Ale volání bez keyword args může bejt napsaný klidně koncentrovaně..